tests/kernels/test_attention.py

```diff
@@ -16,13 +16,17 @@
 NUM_BLOCKS = 40000  # Arbitrary values for testing
 PARTITION_SIZE = 512
 
-DTYPES = [torch.half, torch.bfloat16, torch.float]
+DTYPES = [
+    torch.half,
+    # torch.bfloat16,
+    # torch.float,
+]
 NUM_GEN_SEQS = [7]  # Arbitrary values for testing
 NUM_PREFILL_SEQS = [3]  # Arbitrary values for testing
 NUM_HEADS = [(40, 40), (64, 8)]  # Arbitrary values for testing
-HEAD_SIZES = [64, 80, 96, 112, 128, 256]
-BLOCK_SIZES = [16, 32]
-USE_ALIBI = [False, True]
+HEAD_SIZES = [64]
+BLOCK_SIZES = [16]
+USE_ALIBI = [False]
 SEEDS = [0]
 
```

Comment:
I think we should support the original test cases (DTYPES, HEAD_SIZES, BLOCK_SIZES, USE_ALIBI should be the same).

Reply:
The problem has been fixed

Reply:
Currently we haven't supported bfloat16 in kv cache quant, but other test cases have been restored.